src/constants.ts

```diff
@@ -1,12 +1,12 @@
 import { Edge } from './types';
-import { Connection } from 'twilio-client';
-import { name } from '../package.json';
+import { Call } from '@twilio/voice-sdk';
+import { default as appInfo } from '../package.json';
```

Comment:
I might have been wrong about how we imported package.json in the react app for the app version. 

Here I thought that it would import the whole package.json, and we would just access the .name property. 

But I looked through the whole bundle, and it only includes the name field, and excludes the rest of package.json. This looks just fine. I wonder if we didn't need to bother with the environment variables in the react app?